packages/ckeditor5-dev-tests/lib/utils/automated-tests/parsearguments.js

```diff
@@ -56,6 +56,10 @@ module.exports = function parseArguments( args ) {
 		options.files = options.files.split( ',' );
 	}
 
+	options.language = options.language || 'en';
```

Comment:
I linked to the incorrect file 😞

This is for automated tests, for manual tests we do parsing in the `test-manual.js` file.

Reply:
Manual tests use the same util. See https://github.com/ckeditor/ckeditor5-dev/blob/t%2F529/packages/ckeditor5-dev-tests/bin/test-manual.js#L15 and https://github.com/ckeditor/ckeditor5-dev/blob/t%2F529/packages/ckeditor5-dev-tests/lib/index.js#L11.